pyproject.toml

```diff
@@ -201,35 +201,35 @@ max-line-length = 105  # default 100
 [tool.pylint."messages control"]
 disable = [
 # intentionally disabled:
-    "spelling",  # too noisy
+    "arguments-renamed", # mostly used in the transpiler, renaming seems natural
+    "docstring-first-line-empty", # relax docstring style
+    "duplicate-code", # disabled as it is too verbose
     "fixme", # disabled as TODOs would show up as warnings
+    "import-outside-toplevel", "import-error", # overzealous with our optionals/dynamic packages
+    "nested-min-max", # this gives false equivalencies if implemented for the current lint version
+    "no-else-return",  # relax "elif" after a clause with a return
+    "no-member", # for dynamically created members
+    "no-value-for-parameter", # cannot implement for eldritch horrors: qiskit/providers/options.py
```

Comment:
If this, and `unexpected-keyword-arg`, are just lints that are local to `Options`, could we enable the lints globally and then put a class-level skip on those lints in the relevant place?

Reply:
ready for another look